autoload/slime.vim

```diff
@@ -63,6 +63,28 @@ function! s:KittyConfig() abort
   let b:slime_config["listen_on"] = input("kitty listen on: ", b:slime_config["listen_on"])
 endfunction
 
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+" Wezterm
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+function! s:WeztermSend(config, text)
+	call system("wezterm cli send-text --no-paste --pane-id " . shellescape(a:config["pane_id"]) . " " . shellescape(a:text))
+endfunction
+
+function! s:WeztermConfig() abort
+	if !exists("b:slime_config")
+		let b:slime_config = {"pane_id": 0}
+	end
+	let b:slime_config["pane_id"] = str2nr(system("wezterm cli list --format json | jq '.[][\"pane_id\"]' | tail -n 1"))
```

Comment:
I'm not sure about this: wezterm doesn't guarantee an ordering for the data returned here, and the results might be surprising: for example, if you start with a tab with 2 panes for slime/repl, then create a new tab, this logic will probably select the pane from the new tab which is presumably not the desired outcome.  Connecting or detaching multiplexer domains may have other surprising consequences for the ordering of this data.

It's probably a good idea to take the current pane id from the `$WEZTERM_PANE` environment, then use `wezterm cli list --format json` to discover the tab_id for that pane, then select only the panes from that tab.

If `$WEZTERM_PANE` isn't set, then you could use logic similar to the second bullet in https://wezfurlong.org/wezterm/cli/cli/index.html#targeting-panes to find the least-idle session and use its focused pane id instead.